packages/pn-commons/src/components/NotificationDetail/NotificationDetailDocuments.tsx

```diff
@@ -123,7 +127,7 @@ Props) => {
         </Stack>
       </Grid>
       <Grid key={'download-files-section'} item>
-        {documents && mapOtherDocuments(documents)}
+        {!disableDownloads && documents ? mapOtherDocuments(documents) : []}
```

Comment:
why you don't print the other documents if download is disabled?

Reply:
I fix it because I adapted to the old logic